tests/docker/ducker-ak

```diff
@@ -217,6 +217,8 @@ ask_yes_no() {
 # $1: The name of the image to build.
 ducker_build() {
     local image_name="${1}"
+    local what="${2}"
+    [[ -n "${what}" ]] || what="${image_name}"
```

Comment:
The new argument isn't mentioned in the PR description and isn't used in the rest of the patch.

Reply:
The `what` argument is used when printing the [status](https://github.com/apache/kafka/blob/db688b1a5e5904eed5849fbab4940585e5a3d646/tests/docker/ducker-ak#L241) of the docker build.